pkg/driverbuilder/builder/builders.go

```diff
@@ -254,6 +255,13 @@ func Factory(target Type) (Builder, error) {
 	if strings.HasPrefix(target.String(), "ubuntu") {
 		target = Type("ubuntu")
 	}
+	// For fake "local" target (that is not exposed to users,
+	// nor registered in byTarget, just use "arch".
+	// NOTE: it won't be used anyway, as driverbuilder/local.go
+	// manually creates a "local" builder.
+	if target.String() == "local" {
```

Comment:
This is a workaround to let `local` cmd set `local` target and passing the root options validation step.

Note: `local` builder is not mapped inside the `byTarget` map because i did not want it to be visible by the user when running eg: `docker` builder processor, since it makes no sense.
So, the new `local` builder is hidden and manually instantiated by `local` builder processor.